python/bindings.cpp

```diff
@@ -589,6 +590,109 @@ PYBIND11_MODULE(_libsonata, m) {
         .value("center", SimulationConfig::Report::Compartments::center)
         .value("all", SimulationConfig::Report::Compartments::all);
 
+    py::class_<SimulationConfig::Input> input(m, "Input", "List of parameters of an input");
+    input.def_readonly("module", &SimulationConfig::Input::module, "Type of stimulus")
+        .def_readonly("input_type", &SimulationConfig::Input::input_type, "Type of input")
+        .def_readonly("delay", &SimulationConfig::Input::delay, "Time when input is activated (ms)")
+        .def_readonly("duration",
+                      &SimulationConfig::Input::duration,
+                      "Time duration in ms for how long input is activated")
+        .def_readonly("node_set",
+                      &SimulationConfig::Input::node_set,
+                      "Node set which is affected by input")
+        .def_readonly("amp_start",
+                      &SimulationConfig::Input::amp_start,
+                      "The amount of current initially injected (nA)")
+        .def_readonly("amp_end",
+                      &SimulationConfig::Input::amp_end,
+                      "The final current when a stimulus concludes (nA)")
+        .def_readonly("percent_start",
+                      &SimulationConfig::Input::percent_start,
+                      "The percentage of a cell's threshold current to inject ")
+        .def_readonly("percent_end",
+                      &SimulationConfig::Input::percent_end,
+                      "The percentage of a cell's threshold current to inject at the end")
+        .def_readonly("width",
+                      &SimulationConfig::Input::width,
+                      "The length of time each pulse lasts (ms)")
+        .def_readonly("frequency",
+                      &SimulationConfig::Input::frequency,
+                      "The frequency of pulse trains (Hz)")
+        .def_readonly("percent_less",
+                      &SimulationConfig::Input::percent_less,
+                      "A percentage adjusted from 100 of a cell's threshold current")
+        .def_readonly("spike_file",
+                      &SimulationConfig::Input::spike_file,
+                      "The location of the file with the spike info for injection")
+        .def_readonly("source",
+                      &SimulationConfig::Input::source,
+                      "The node set to replay spikes from")
+        .def_readonly("voltage",
+                      &SimulationConfig::Input::voltage,
+                      "The membrane voltage the targeted cells should be held at (mV)")
```

Comment:
Once #201 is in, I think it's best if we use the same style here; that way we don't need to repeat the string from the structs.

Reply:
thanks!

Reply:
Done